internal/app/service/service_handler_events.go

```diff
@@ -109,3 +111,41 @@ type RemoveIssueRepositoryFromServiceEvent struct {
 func (e *RemoveIssueRepositoryFromServiceEvent) Name() event.EventName {
 	return RemoveIssueRepositoryFromServiceEventName
 }
+
+// OnServiceCreate is a handler for the CreateServiceEvent
+// Is creating a single default priority for the default issue repository
+func OnServiceCreate(db database.Database, event CreateServiceEvent) {
+	//@todo configure somewhere
+	var defaultPrio int64 = 100
```

Comment:
we could define this as environment variables in the config

Reply:
Implemented in PR https://github.com/cloudoperators/heureka/pull/209/files#diff-6b3cba3b631a235ca248056c4c9624ecb1b7c0b5811bcc12af7261e9731dbc77